astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Combinging SSO and MFA in FusionAuth
+description: In this article, you'll learn how to combine single sign-on with multi-factor authentication in FusionAuth.
+author: Matt Keib
+section: Authentication
+tags: single sign on mfa multi factor authentication sso security
+icon: /img/icons/sso-mfa.svg
+darkIcon: /img/icons/sso-mfa-dark.svg
+---
+
+No matter how strong a password is, if an attacker wants to access what lies behind it---data, administrator access to a database or an admin panel, you name it---they eventually will.
+
+Yet there's an equally well-known fact: there's no security policy end users will not complain about, no matter how easy to implement it may be or how important the measures taken are. On top of that, the executive board always seems to side with the users instead of backing [DevSecOps](https://www.redhat.com/en/topics/devops/what-is-devsecops) and system administrators, who want to safeguard the organization and improve its security posture.
```

Comment:
We have 0 internal links in this article at the top. I think it's important for internal backlinks. Can you add some?

Reply:
added some.